src/components/AssetDownload.tsx

```diff
@@ -26,7 +26,7 @@ interface IPropsWithSVG extends IPropsBase {
 }
 interface IPropsWithImage extends IPropsBase {
   svg?: never
-  image: string
+  image: Record<string, unknown>
```

Comment:
@pettinarip since this prop is supposed to receive the GatsbyImageData object i changed this type to match any object. It was set to string before and it was a pain to make it work when passing a Gatsby auto-generated type (from the Queries namespace).

Reply:
This is nice! as commented in the other comment, we will leave it as `any` for now. Once we have better types from gatsby we will refactor and fix all the images types 👍🏼